service/templates/admin.html

```diff
@@ -954,7 +954,8 @@ <h2 class="modal-title">[[modalTitle]]</h2>
               </md-card-title>
               <md-card-content>
                 <md-input-container class="control">
-                  <textarea ng-model="tempPlugins" rows="3" md-select-on-focus></textarea>
+                  <textarea ng-model="tempPlugins" rows="3" md-select-on-focus
+                   style="font-family:courier new, courier, monospace;"></textarea>
```

Comment:
I'd recommend just setting it to monospace, and letting the user's browser prefs dictate the font, but lgtm.